examples/libs/polkadot/broadcast/main.ts

```diff
@@ -0,0 +1,93 @@
+const initDfnsWallet = async (walletId: string) => {
+  const signer = new AsymmetricKeySigner({
+    privateKey: process.env.DFNS_PRIVATE_KEY!,
+    credId: process.env.DFNS_CRED_ID!,
+    appOrigin: process.env.DFNS_APP_ORIGIN!,
+  })
+
+  const dfnsClient = new DfnsApiClient({
+    appId: process.env.DFNS_APP_ID!,
+    authToken: process.env.DFNS_AUTH_TOKEN!,
+    baseUrl: process.env.DFNS_API_URL!,
+    signer,
+  })
+
+  const senderWallet = await DfnsWallet.init({
+    walletId: walletId,
+    dfnsClient,
+  })
+
+  return { senderWallet, dfnsClient }
+}
+
+async function main() {
+  const polkadotWalletId = process.env.POLKADOT_WALLET_ID!
+
+  // Sender wallet is only used here to retrieve the wallet address
+  const { senderWallet, dfnsClient } = await initDfnsWallet(polkadotWalletId)
+
+  const httpProvider = new HttpProvider(process.env.POLKADOT_NODE_URL!);
+
+  // In polkadot, an account must have at least 1 DOT, we need to make sure that
+  // this address has funds or that we transfer at least 1 DOT.
+  // https://wiki.polkadot.network/docs/learn-accounts#existential-deposit-and-reaping
+  const receiverAddress = '5DLJur1FsXezqiRvsq7nTJGDGszDW4xtNeENAYBMXPwPY9bZ'
+
+  // 1 WND or DOT, precision of the asset is 12
+  const amount = 10 ** 12;
+
+  console.log(`Sending ${amount / (10 ** 12)} DOT/WND to the address ${receiverAddress}`);
+
+  const { block: { header: { number: blockNumber } } } = await httpProvider.send('chain_getBlock', []) as any
+  const blockHash = await httpProvider.send('chain_getBlockHash', []) as string
+  const genesisHash = await httpProvider.send('chain_getBlockHash', [0]) as string
+  const metadataRpc = await httpProvider.send('state_getMetadata', []) as `0x${string}`
+  const nonce = await httpProvider.send('system_accountNextIndex', [senderWallet.address]) as number
+  const { specVersion, transactionVersion, specName } = (await httpProvider.send('state_getRuntimeVersion', [])) as any
+
+  const registry = getRegistry({ chainName: 'Westend', specName, specVersion, metadataRpc, });
+
+  const unsigned = methods.balances.transferKeepAlive(
+    {
+      value: amount.toString(),
+      dest: { id: receiverAddress },
+    },
+    {
+      address: senderWallet.address,
+      blockHash: blockHash,
+      blockNumber: parseInt(blockNumber, 16),
+      eraPeriod: 64,
+      genesisHash: genesisHash,
+      metadataRpc: metadataRpc,
+      nonce: nonce,
+      specVersion: specVersion,
+      tip: 0,
+      transactionVersion: transactionVersion,
+    },
+    {
+      metadataRpc: metadataRpc,
+      registry: registry
+    },
+  );
+
+  const signerPayload = registry.createType('SignerPayload', unsigned, { version: unsigned.version })
+
+  // Broadcast transaction
+  const response = await dfnsClient.wallets.broadcastTransaction({
+    walletId: polkadotWalletId,
+    body: { kind: 'Transaction', transaction: signerPayload.toHex() },
+  })
+  console.log(`Transaction submitted with hash ${response.txHash}`)
+}
```

Comment:
let's drop the broadcast example too. instead, we will strip this down as much as possible and add to our docs (like [here](https://docs.dfns.co/d/api-docs/wallets/broadcast-transaction-from-wallet/solana-broadcast-transaction)).